### DISCUSSION TITLE
[Question] GUI Clients for WinGet

### DISCUSSION THREAD

**Discussion by marcelwgn** [microsoft/winget-cli#371] [General]
While WinGet is great for professionals, people not so familiar with the command line often prefer GUI clients.

My question is: Are third party GUI clients for WinGet allowed? Are there any plans to provide a first party client for WinGet?

**Comment by denelon** [comment]
A couple of different projects have already been released providing either a GUI or web-based UI. I think the challenge is how frequently we need to make potentially breaking changes to the client as we evolve towards a 1.0 release. We don't currently (as of 0.1) have a non command line UI on the backlog.

**Comment by doctordns** [comment]
Without cmdlets, a 1.0 release is less than interesting to IT Pros. WHY won't the team admit their mistakes, refactor the code, and move on with a product that IT pros could love? I like the idea of a good package manager that can help organisations manage packages - this tool is not (yet) it.  Unless the intention is to just produce another power toy for some developers to play with.
